### ISSUE TITLE
ships can be added after deployment is locked

### ISSUE THREAD

**Issue by boredompwndu** [danbuis/full_thrust_pbp#49]
![image](https://user-images.githubusercontent.com/8730661/221731808-6d8cb95a-081f-4c01-9c6f-46c5dd7acaab.png)

**Comment by boredompwndu** [comment]
same game as #48

**Comment by danbuis** [comment]
Yeah. the dialog box will change to allow ships to arrive via FTL.  This one isn't surprising
